### ISSUE TITLE
Open for contributions?

### ISSUE THREAD

**Issue by larshp** [SAP-docs/abap-platform-trial-image#26]
Hi,
What is the policy on contributions and issues in this repository?

it currently looks like both issues and pull requests are actively being ignored? which is fine, but I suggest updating it in the documentation so expectations are aligned, https://github.com/SAP-docs/abap-platform-trial-image/issues/14

**Comment by julieplummer20** [comment]
Hi Lars,
I'm really sorry, I thought I had replied much earlier.
However, I do work through the Open Issues in this Repo, according to capacity.
I would actually welcome any feedback from you or anyone, via this channel. Some users have already done so; I will also be checking it more frequently this year and have updated my notification settings accordingly.
We want to release 2023 asap this quarter, so I will publish this repo address with a new request for feedback; hopefully, that will boost interest.
TL;DR - yes, please feel free to provide any feedback, right here. Thanks for your support and apologies for the very late response.
Happy New Year!
Julie.

**Comment by larshp** [comment]
thanks

happy new year 🥳 

closing

**Comment by julieplummer20** [comment]
Great - could you also please close [#14](https://github.com/SAP-docs/abap-platform-trial-image/issues/14), if you're ok with that?
I'm just working through the backlog after Christmas now.
Many thanks and best wishes,
Julie.
